dotCMS/src/main/java/com/dotcms/exception/ExceptionUtil.java

```diff
@@ -457,6 +458,19 @@ public static String getCurrentStackTraceAsString (final int limit) {
         return getStackTraceAsString(limit, traces);
     }
 
+    /**
+     * Checks whether a specific method name with its fully-qualified class name has been called
+     * at least X number of times.
+     * @param methodName
+     * @param maxCount
+     * @return
+     */
+    public static boolean isMethodCallCountGteThan(final String methodName, final int maxCount) {
```

Comment:
I would expect this method to be in the `ThreadUtil` I think.

Reply:
Done!